workflows/cellbender/cellbender.wdl

```diff
@@ -0,0 +1,180 @@
+version 1.0
+
+workflow cellbender {
+  input {
+      # Sample name
+      String sample_name
+      # cellranger h5 file
+      File input_10x_h5_file_or_mtx_directory
+      # Outputs
+      String output_directory
+      # Docker image for cellbender remove-background version
+      String docker_registry = "quay.io/cumulus"
+      # Cellbender version to use. Currently support: 0.2.0
+      String cellbender_version = "0.2.0"      
+
+      # Expected cells
+      Int? expected_cells
+      # The number of droplets from the rank-ordered UMI plot that will be analyzed
+      Int? total_droplets_included
+      # Which model is being used for count data. ["simple", "ambient", "swapping", "full"]
+      String? model
+      # Droplets with UMI counts below this number are completely excluded from the analysis
+      Int? low_count_threshold
+      # Target false positive rate in (0, 1).  A false positive is a true signal count that is erroneously removed.
+      String? fpr
+      # Number of epochs to train.
+      Int? epochs
+      # Dimension of latent variable z.
+      Int? z_dim
+      # Dimension of hidden layers in the encoder for z.
+      String? z_layers
+      # Training detail: the fraction of the training data each epoch that is drawn (randomly sampled) from surely empty droplets.
+      Float? empty_drop_training_fraction
+      # Integer indices of genes to ignore entirely.
+      String? blacklist_genes
+      # Training detail: lower learning rate for inference.
+      Float? learning_rate
+      # Cause remove-background to operate on gene counts only, ignoring other features.
+      Boolean exclude_antibody_capture = false
+
+      # Google cloud zones to consider for execution
+      String zones = "us-central1-a us-central1-b us-central1-c us-central1-f us-east1-b us-east1-c us-east1-d us-west1-a us-west1-b us-west1-c"
+      # Disk space in GB needed per sample
+      Int disk_space = 50
+      # Boot disk space in GB needed per sample
+      Int boot_disk_size_GB = 20
+      # Number of maximum preemptible tries allowed
+      Int preemptible = 2
```

Comment:
We may set preemptible to `0` to be consistent with the official workflow, especially given that this workflow uses GPU machines by default.